scripts/fao/preprocess.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import csv
+import pycountry
+
+# Open the original .csv file for reading
+with open('ExchangeRate_Currency.csv', 'r', encoding='ISO-8859-1') as csv_in:
+    # Open the new .csv file for writing
```

Comment:
nit: it would be good to do this by creating functions even though in this case you can get away with not creating functions. https://google.github.io/styleguide/pyguide.html#383-functions-and-methods